src/content/docs/release-notes/docs-release-notes/docs-7-19-2024.mdx

```diff
@@ -0,0 +1,43 @@
+---
+subject: "Docs"
+releaseDate: '2024-07-19'
+version: 'version: July 12-18, 2024'
+---
+
+### New docs
```

Comment:
Why are we using h3s instead of h2s?

Reply:
Not sure. It's what all of our release notes use, so I just followed the pattern.